README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/mimblewimble/grin.svg?branch=master)](https://travis-ci.org/mimblewimble/grin) [![Gitter chat](https://badges.gitter.im/grin_community/Lobby.png)](https://gitter.im/grin_community/Lobby)
+[![Build Status](https://travis-ci.org/mimblewimble/grin.svg?branch=master)](https://travis-ci.org/mimblewimble/grin) [![Gitter chat](https://badges.gitter.im/grin_community/Lobby.png)](https://gitter.im/grin_community/Lobby) [![Codecov coverage status](https://codecov.io/gh/mimblewimble/grin/branch/master/graph/badge.svg)](https://codecov.io/gh/mimblewimble/grin)
```

Comment:
Badge also won't work until `mimblewimble/kcov` exists, so a report can be uploaded for `mimblewimble/grin`.

Reply:
@sdtsui Is there some access rights that need to be arranged, or how is graph/badge.svg getting updated?
(Also, kcov now builds for me locally, so I can help more in testing this)